cli50/__main__.py

```diff
@@ -54,6 +54,19 @@ def main():
     except subprocess.TimeoutExpired:
         sys.exit("Docker not responding.")
 
+    # Check for newer version
+    try:
+        latest = max(requests.get("https://pypi.org/pypi/cli50/json").json()["releases"])
+        assert latest <= __version__
+    except requests.RequestException:
+        pass
+    except AssertionError:
+        print(
+            "You are using cli50 version {}, however version {} is available.".format(__version__, latest),
+            "You should consider upgrading via the 'pip3 install --upgrade cli50' command.",
+            sep='\n'
+        )
```

Comment:
We might want to shorten this output a bit, but, even so, do use `_`, a la

```
print(_("..."))
```

so that it can be localized eventually in other languages.

Reply:
Per https://github.com/cs50/cli50/pull/79/commits/7a54f89c99a7f5a7c52262a200ab780d24e65495 I have shorted the message but feels it needs more elaborating! Perhaps include the instructions on updating?